.github/workflows/release-rp-storage-tool.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           - release_for: linux-amd64
             os: ubuntu-latest
           - release_for: linux-arm64
-            os: ubuntu-latest-2-arm64
+            os: ubuntu-latest-4-arm64
```

Comment:
there is one at org level 

![Screenshot 2024-09-22 at 11 08 19](https://github.com/user-attachments/assets/19c80935-2a8f-4380-a8c7-ce209408bf39)

the runner is available to all repos in the org. To save $$ i think we should leave the 2-core here

Reply:
i believe i found the cause of the [failures](https://github.com/redpanda-data/redpanda/actions/workflows/release-rp-storage-tool.yml): `ubuntu-latest-2-arm64` was not enabled for public repos at the org level. i enabled it and it now [shows up at the repo level](https://github.com/redpanda-data/redpanda/actions/runners).

i'll drop commit 9b2f899 from this PR so we can continue to use 2-arm64